### ISSUE TITLE
twilio@4.0.1: decompress error

### ISSUE THREAD

**Issue by rfks** [twilio/scoop-twilio-cli#11]
PS C:\Users\rfks\Downloads> scoop install twilio
Installing '7zip' (22.01) [64bit] from main bucket
7z2201-x64.msi (1,8 MB) [=====================================================================================] 100%
Checking hash of 7z2201-x64.msi ... ok.
Extracting 7z2201-x64.msi ... done.
Linking ~\scoop\apps\7zip\current => ~\scoop\apps\7zip\22.01
Creating shim for '7z'.
Creating shortcut for 7-Zip (7zFM.exe)
Persisting Codecs
Persisting Formats
Running post_install script...
'7zip' (22.01) was installed successfully!
Notes
-----
Add 7-Zip as a context menu option by running: "C:\Users\rfks\scoop\apps\7zip\current\install-context.reg"
Installing 'twilio' (4.0.1) [64bit] from twilio-scoop bucket
twilio-win32-x64.tar.xz (32,3 MB) [===========================================================================] 100%
Checking hash of twilio-win32-x64.tar.xz ... ok.
Extracting twilio-win32-x64.tar.xz ... ERROR Exit code was 2!
Failed to extract files from C:\Users\rfks\scoop\apps\twilio\4.0.1\\twilio-win32-x64.tar.
Log file:
  ~\scoop\apps\twilio\4.0.1\\7zip.log

Please try again or create a new issue by using the following link and paste your console output:
https://github.com/twilio/scoop-twilio-cli/issues/new?title=twilio%404.0.1%3a+decompress+error

**Comment by shrutiburman** [comment]
Hi @rfks , Thanks for reaching out. We've identified this issue internally and are working on it. Meanwhile, to unblock you can download the updated cli version from releases executables present [here](https://github.com/twilio/twilio-cli/releases/tag/4.0.1#:~:text=12%20days%20ago-,twilio%2D4.0.1.exe,-46.3%20MB).

**Comment by rfks** [comment]
cheers @shrutiburman !

**Comment by shrutiburman** [comment]
Hi @rfks ,
This has been fixed in our latest release 🚀.
Closing this issue, feel free to re-open if you're still facing this. 
![Screenshot 2022-08-25 at 12 16 06 PM](https://user-images.githubusercontent.com/87537688/186598439-88aa8cdc-ae47-4326-a947-eb9f823aa2ae.png)
